bazarr.py

```diff
@@ -150,23 +128,30 @@ def daemon(bazarr_runner=lambda: start_bazarr()):
             else:
                 bazarr_runner()
 
+    def should_stop():
+        return daemonStatus.should_stop()
 
-    bazarr_runner = lambda: start_bazarr()
```

Comment:
The duplicate declarations were only required when having to support Python 2 and Python 3. Since here it's no longer the case we can have a single definition.

And therefore we can turn those variable declarations into mere function declarations.